### PR TITLE
Feature/pelagos 3785 add dataset monitoring

### PR DESCRIPTION
**Pull request by praneethpr**
Dataset Monitoring and Dataland should work in this pull request

### CODE REVIEW

**Comment by mickel1138** [comment]
We fixed the API issues with this info:
https://stackoverflow.com/questions/30114152/fosrestbundle-setup-for-return-json-but-still-asking-for-twig-template

File: .env [hunk 1]
```diff
@@ -52,3 +52,4 @@ LDAP_PEOPLE_OU='ou=users,dc=domain,dc=org'
 ACCOUNT_LESS_STRICT_PASSWORD_RULES=FALSE
 MAILER_FROM_ADDR='email@domain.null'
 MAILER_FROM_NAME='SenderName'
+PELAGOS_NAME=pelagos
```
Comment (mickel1138) [inline .env]: This is just a weird parameter. Is this different in env.local?

Comment (praneethpr) [reply .env]: The value is the pelagos name of our instance, which you need to add in the .env.local file
For example:
PELAGOS_NAME=pelagos.ppondicherry

File: config/routes/annotations.yaml [hunk 2]
```diff
@@ -1,3 +1,5 @@
 controllers:
     resource: ../../src/Controller/
     type: annotation
+    options:
+        expose: true
```
Comment (mickel1138) [inline config/routes/annotations.yaml]: This doesn't seem to make any difference? Where does it break if you change this?

File: config/packages/fos_elastica.yaml [hunk 3]
```diff
@@ -3,4 +3,70 @@ fos_elastica:
     clients:
         default: { host: localhost, port: 9200 }
     indexes:
-        app: null
+        pelagos:
```
Comment (mickel1138) [inline config/packages/fos_elastica.yaml]: Should we call this 'app'?

File: composer.json [hunk 4]
```diff
@@ -77,7 +77,8 @@
     "scripts": {
         "auto-scripts": {
             "cache:clear": "symfony-cmd",
-            "assets:install %PUBLIC_DIR%": "symfony-cmd"
+            "assets:install %PUBLIC_DIR%": "symfony-cmd",
+            "fos:js-routing:dump --format=json --target=public/js/fos_js_routes.json": "symfony-cmd"
```
Comment (mickel1138) [inline composer.json]: we should use %PUBLIC_DIR% ?

File: config/routes/annotations.yaml (hunk 2 above)
Comment (praneethpr) [reply config/routes/annotations.yaml]: Try running the fos:js-routing:dump/debug, it should be empty without that config

File: composer.json (hunk 4 above)
Comment (praneethpr) [reply composer.json]: Will add it to the wiki List

File: config/routes/annotations.yaml (hunk 2 above)
Comment (mickel1138) [reply config/routes/annotations.yaml]: I can't seem to make it fail?

File: config/packages/fos_elastica.yaml (hunk 3 above)
Comment (praneethpr) [reply config/packages/fos_elastica.yaml]: Will add it to the wiki List

**Pull request merged**